js/page-specific/branches-filter.js

```diff
@@ -71,37 +77,23 @@ bcpl.pageSpecific.filter = (($) => {
 		console.log('err', errorThrown);
```

Comment:
Should this be gone?

Reply:
We need a good idea for client-side error handling first.